src/appmixer/intelswift/core/HookCreate/HookCreate.js

```
@@ -0,0 +1,33 @@
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { platformType, syncId, name, firstName, lastName, tenantId, projectUUID } = context.messages.in.content;
+
+        // remaining to handles tenantId and projectId
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)